app/controllers/provider_foreman_controller.rb

```diff
@@ -583,6 +612,34 @@ def configuration_profile_node(id, model)
     end
   end
 
+  def inventory_group_node(id, model)
+    @record = @inventory_group_record = if model
+                                          find_record(ManageIQ::Providers::ConfigurationManager::InventoryGroup, id)
+                                        else
+                                          ManageIQ::Providers::ConfigurationManager::InventoryGroup.new
```

Comment:
Why is an `InventoryGroup` being built here?  They should be pulled in from refresh.